### PR TITLE
feat: Apl-248 github actions update to support valid dns by default

### PR DESCRIPTION
**Pull request by Ani1357**
This PR updates our integration github actions to use valid domain names be default.
Key changes in this PR:
- removed Scaleway config
  -  deleted `.github/workflows/integration-on-schedule-dns-no-admin-pass.yml`
- removed Azure config
- Removed nip.io configs
  - removed `.github/workflows/integration-on-schedule-dns-no-admin-pass.yml`
  - removed `.github/workflows/integration-on-schedule-dns.yml`
  - updated nightly integration tests for `full`, `upgrade`, `minimal`, `minimal-with-teams` and `minimal-no-admin-pass`
- Set default `generate_password: yes`
- Set default `kms: age`
- Set default `certificate: letsencrypt_production`

### Testing:

- From [Deploy Otomi github action tab](https://github.com/linode/apl-core/actions/workflows/integration.yml) click **Run Workflow**.  
- In **Use workflow from** select **Branch: APL-248_gha-update**. 
- Select your **domain** and click **Deploy**. 
- After the deployment finishes you can download the kubeconfig and monitor the argocd applications status until they are ready.
- When ready you can access your cluster with the randomly generated password.

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-api: main
-console: main
-tasks: main
+api: APL-132-prometheus-ha
```
Comment (j-zimnowoda) [inline versions.yaml:1]: Could you revert versions ?

Comment (Ani1357) [reply versions.yaml]: updated

**Review by j-zimnowoda — APPROVED** [review]
Tested with my domain. Works well

**Review by ElderMatt — CHANGES_REQUESTED** [review]
If this is possible it would be a nice change.

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -77,107 +58,83 @@ on:
...
-      dns:
+      domain_alias:
         type: choice
-        description: Select DNS provider
+        description: Select Domain Alias
         options:
-          - nip_io
-          - az_dns
-        default: nip_io
+          - select_your_domain_alias
```
[the remark this answers is not in the document]
Comment (j-zimnowoda) [reply .github/workflows/integration.yml:65]: wow, great idea!

**Review by j-zimnowoda — APPROVED** [review]

**Review by ElderMatt — APPROVED** [review]

**Pull request merged**